pcdsdevices/slits.py

```diff
@@ -77,7 +78,7 @@ def __init__(self, prefix, *, slit_type="", name=None,
 
     @property
     def egu(self):
-        """Engineering units"""
+        """EnGineering Units."""
```

Comment:
?

Reply:
Is that what it stands for?
I saw some other classes use weird capitalization like that so I did it here too but its ugly so I'll remove it.

Reply:
It is what EGU stands for, as awful as that is. ~Why not `.UNIT`, EPICS gods?~

Reply:
Ohhhh, I understand why there were funky caps now. I thought it was a typo.